frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -66,6 +67,14 @@ export function SDKs({
         }
     }, 2000)
 
+    const HelpCard = (): JSX.Element => (
+        <LemonCard className={`${isUserInNonTechnicalTest ? 'mb-2' : 'mt-6'}`} hoverEffect={false}>
+            <h3 className="font-bold">Need help with this step?</h3>
```

Comment:
Less related but maybe we can change this to "Need help?" so it fits better along the first line of the card.